packages/react/src/box/styles.ts

```diff
@@ -387,12 +387,18 @@ export const linkStyles = {
 type FocusProps = Partial<{
 	/** If true, the focus indicator will be shown when the element receives keyboard focus. */
 	focus: boolean;
+	/** If true, the focus indicator will be shown when the element receives keyboard and programmatic focus. */
+	focusAll: boolean;
 }>;
 export const focusStyles = {
 	':focus': packs.outline,
 	':focus:not(:focus-visible)': { outline: 'none' },
 	':focus-visible': packs.outline,
 };
+export const focusAllStyles = {
+	...focusStyles,
+	'&:focus:not(:focus-visible)': packs.outline,
+};
```

Comment:
Feels weird to spread `focusStyles` when what you really want is just `':focus': packs.outline`. If we make focus/focusAll either/or, then perhaps the focusStyles could selectively apply the `:focus-visible` styles?

Reply:
We have a bunch of components with `focus` already applied. I feel this is the safest "next step", despite it's oddness. Let me explore a bit and get back to you.

Reply:
It would be no different afaict, effectively compiling to:

`const focusStyles = focus ? focusStyles : focusAll ? focusAllStyles : undefined`